docs/api/layouts/index.md

```diff
@@ -23,9 +23,18 @@ certain way.
 
   marimo.accordion
   marimo.callout
-  marimo.image
   marimo.hstack
   marimo.vstack
   marimo.tabs
   marimo.tree
 ```
+
+There are also a set of utility elements that make it easier to build HTML components.
```

Comment:
Good call ... later, when we have `mo.video` and `mo.ui.webcam`, we can split these out into another page.